install/preview/prettylog/main.go

```diff
@@ -23,7 +23,7 @@ var msgs = []struct {
 	{Msg: "preparing system", Success: "extracting images to download ahead"},
 	{Msg: "downloading images", Success: "--output-split-files"},
 	{Msg: "preparing Gitpod preview installation", Success: "rm -rf /var/lib/rancher/k3s/server/manifests/gitpod"},
-	{Msg: "starting k3s", Success: "ws-proxy"},
```

Comment:
I think `k3s` is an implementation detail that we don't have to show it to the user here